packages/odyssey/src/scss/base/_typography-global.scss

```diff
@@ -10,8 +10,10 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+@use "sass:math";
+
 html {
-  font-size: ($base-font-size / 16px) * 100%; // converts px value to % of browser default
+  font-size: math.div($base-font-size, 16px) * 100%; // converts px value to % of browser default
```

Comment:
can this be done with css calc?

Reply:
Sadly, not yet. 😞 CSS3 `calc()` doesn't support division via units, only via whole numbers. It's also not safe to use within our MS functions elsewhere because the output remains a calc expression, not a string or number.

We could probably refactor our modular scale to avoid the use of units, but it'd take some deeper rejiggering.

Reply:
hmmm ok something we can work toward in the future